### ISSUE TITLE
Fix new flake8 error

### ISSUE THREAD

**Issue by vfdev-5** [pytorch/ignite#421]
Impacts `master`
https://travis-ci.org/pytorch/ignite/jobs/486833029#L466
```
$ flake8
./ignite/metrics/accuracy.py:42:17: E117 over-indented
./tests/ignite/metrics/test_metrics_lambda.py:11:9: E117 over-indented
```

and `nightly`
https://travis-ci.org/pytorch/ignite/jobs/486949548#L452

**Comment by anmolsjoshi** [comment]
Fixed via latest commit in #420 

Lines were actually over indented.

**Comment by anmolsjoshi** [comment]
@vfdev-5 sending in a PR for this.
